applications/LiisaPortfolioTracker.md

```diff
@@ -0,0 +1,279 @@
+
+
+### Team Code Repos
+
+Source codes will reside in
+- https://github.com/LiisaNFT
+
+For further reference
+- https://github.com/paulogustavopeixoto
+- https://github.com/guilhermeleitao2002
+- https://github.com/kerimtumkaya
+- https://github.com/alexcollignon
+
+### Team LinkedIn Profiles 
+
+- https://www.linkedin.com/in/bernardo-silva-631149161/
+- https://www.linkedin.com/in/pauloassispeixoto/
+- https://www.linkedin.com/in/kerim-caner-tumkaya/
+- https://www.linkedin.com/in/guilherme-leit%C3%A3o-47bb27192/
+- https://www.linkedin.com/in/alexandre-collignon/
+
+## Development Status :open_book:
+
+Our team has brought the first iteration of the portfolio tracker mockups to fruition (please see them below in the Mockups section).
+
+In a proactive endeavor to understand and address the data-related challenges of building the MVP, we have exchanged several emails and engaged in multiple meetings with the Parity team. Through these communications, we ensured technical feasibility of the proposed approach and refined the data approach and mockups to better match the ecosystems' needs.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):**  2.25
+- **Total Costs:** 30,000 USDC
+
+### Milestone 1 — On-chain data collection, indexing and calculations
+
+- **Estimated duration:** 1.5 months
+- **FTE:**  2.5
+- **Costs:** 17,000 USDC
+
+| Number | Deliverable               | Specification |
+| ------ | ------------------------- | ------------- |
+| 0a.    | License                   | All code will be open-sourced under Apache 2.0 license. |
+| 0b.    | Documentation             | Documentation includes Inline Code Documentation, Configuration Documentation, Readme file. Documentation on the modular design of Subsquid API calls is included. |
+| 0c.    | Testing Guide             | The code will have unit-test coverage (min. 50%) to ensure functionality and robustness. In the guide, we will describe how to run these tests |
+| 0d.    | Docker                    | Provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1a.    | Modular Subsquid API calls | Design and implement API queries to extract event data from the Subsquid API for the supported protocols: Astar, Basilisk, Efinity, Moonbeam, Unique Network, Moonriver  and Moonsama. The design will be modular to allow easy replacement with different APIs. This will be developed using Node.js. |
```

Comment:
> Design and implement API queries to extract event data from the Subsquid API for the supported protocols: Astar, Basilisk, Efinity, Moonbeam, Unique Network, Moonriver and Moonsama. The design will be modular to allow easy replacement with different APIs. This will be developed using Node.js.

Are you going to support all NFTs on these networks? No matter what standards or pallet's they'd use? Please clarify directly within the Spec here.

Reply:
Dear @semuelle ,

Thank you for your insightful feedback. We agree that scoping the project clearly is critical for its success. To address your question about supporting all NFTs on the networks we listed, we have made specific choices in line with our platform's capabilities and the project budget.

For our initial MVP, we will support the following NFT standards for each protocol:

Astar Network: ERC-721, ERC-1155, PSP 34 (similar to ERC-721), and PSP 37 (similar to ERC-1155)

Basilisk: We will support the Uniques pallet, which is straightforward and standardized.

Efinity: ERC-721 and ERC-1155. While Efinity does support additional standards, we will initially focus on these two.

Moonbeam: ERC-721 and ERC-1155.

Unique Network: We'll support their native NFTs.

Moonriver: ERC-721 and ERC-1155.

Moonsama: ERC-721 and ERC-1155.

The reason for these choices is mainly that our platform is already compatible with ERC-721 and ERC-1155 standards. The other pallets and standards we selected are either identical or similar enough that we can integrate them easily.

We hope this clarifies our approach for the MVP. As the project evolves, we are certainly open to expanding the range of supported standards based on community demand and technical feasibility.

Thank you once again for your thoughtful questions; they help make our project better.

Best regards,
Liisa Team

Reply:
Lovely, thanks for the update.